src/app.md

```diff
@@ -271,6 +272,40 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-file-skip-dot]: #ex-file-skip-dot
+<a name="ex-file-skip-dot"></a>
+##  Traverse directories while skipping dotfiles
+
+Iterates over filesystem entries efficiently skipping hidden files and directories with help of [WalkDir::filter_entry]. 
```

Comment:
I see that the first paragraph doesnt have the string inside the brakets... now I fixed in another way (compeltely removing this extra paragraph). I just commit.